website/src/controllers/api-v2/groups.js

```diff
@@ -691,8 +691,9 @@ var inviteByEmails = function(invites, group, req, res, next){
 
 api.invite = function(req, res, next){
   var group = res.locals.group;
-
-  if (req.body.uuids) {
+  if (group.memberCount + group.invites.length >= 30) {
+    return res.json(400, {err: "Over 30 member party cap"});
```

Comment:
Also the error message (in fact, any text anywhere that the user can see) should be translatable. I'm happy to give more details about how to do this, if you wish, including how to include `30` as a variable in it - just ask if you want suggestions (and make me write that up in the Blacksmiths wiki page!)